config/clients/python/template/README_calling_api.mustache

```diff
@@ -0,0 +1,11 @@
+{{#apiInfo}}
```

Comment:
Let's write this by hand. It is not too useful otherwise. Use the same examples as the other SDKs for consistency

Reply:
Done